nautobot/core/templatetags/buttons.py

```diff
@@ -203,7 +203,7 @@ def consolidate_bulk_action_buttons(context):
                 attrs=render_tag_attrs(
                     {
                         "class": edit_button_classes,
-                        "formaction": reverse(context["bulk_edit_url"]) + params,
+                        "formaction": reverse(context["bulk_edit_url"]),
```

Comment:
Does *seem* like a similar change would be appropriate at line 245 as well. Can we add a test case or two for this issue?

Reply:
potentially. That is what I was going for with `Argueably they should be removed from delete params as well`.  Not sure it would ever matter on bulk delete though. Thoughts?

Reply:
If they're not needed, let's remove them there as well.

Reply:
I think so yeah.

Reply:
removed and added a test case. There is not an easy way to show that a url can still be added to the bulk edit, but this shows it is not copied over. 

Removed params from bulk edit as well, but this was not actually an issue, and thus nothing to test.